### ISSUE TITLE
Validation on mandatory properties not working and icon-alert constantly showing

### ISSUE THREAD

**Issue by prgriffithsdev** [skttl/our-umbraco-matryoshka#16]
Hi, 

I have just run the following **Install-Package Our.Umbraco.Matryoshka -Version 0.2.0** and the validation on mandatory properties is not working. 

![13](https://user-images.githubusercontent.com/56939169/79836804-8ce84c80-83a8-11ea-8ad9-a19545eb2120.png)

Also, on the tabs, the validation the alert icon (<i class="icon icon-alert"></i>) always diplays as if validation has already triggered ?

![12](https://user-images.githubusercontent.com/56939169/79836790-8954c580-83a8-11ea-9303-7d1efafd2d28.png)

I think validation of mandatory properties was and issue but the icon issue wasn't a problem in the previous versions so it was something that was changed recently. I think its related to the validation css in 

App_Plugins\Our.Umbraco.Matryoshka\tabbed-content.css

Im unsure if this is getting picked up? 

```
.matryoshka-tabs-list .umb-tab i.icon {
    display: none;
}
```
thanks 
Paul

**Comment by skttl** [comment]
Just tried a clean Umbraco 8.6 and Matryoshka 0.2, but can't reproduce. Can you provide steps to reproduce?

**Comment by prgriffithsdev** [comment]
Hey @skttl, 

- I was running Umbrcao 8.5.4 with Matryoshka 0.1.0 installed (via the back office)
- Upgraded to Umbraco 8.6 and noticed the media issue 
- Applied the css fix which resolved the media select issue. Continued working  
- Added a new custom dropdown (with prevalues) property to a doc type and defined as mandatory
- Created an instance of the doc type and when selecting save and Publish the validation is being bypassed despite having two dropdown mandatory properties on the doc type
- Also, noticed the i.icon alert issue 
- Decided to fully remove  the Matryoshka 0.1.0 (with the css addition) package. Unistalled from the umbraco backoffice and deleted the references from App_Pluggins
- Reinstalled Matryoshka 0.2 via Nuget (https://www.nuget.org/packages/Our.Umbraco.Matryoshka)
- Issue still present despite deleting all the TEMP folder

Is there anything else I can try?

Thank you for looking

Paul

**Comment by skttl** [comment]
Thanks for getting back :)

Try bumping the version number in config/clientdependency.config, and then a hard reload in your browser (in Chrome, open dev tools, right click the reload button, select hard reload)

**Comment by prgriffithsdev** [comment]
Hi @skttl,

I feel so bad right now! Hard reload solves the issue!! FML. I was working on this late last night and it was obviously time to call it a day! You have to love chrome.

Apologies for raising this issue but thank you for being supportive and responding quickly. 

This package is great, thank you for developing and sharing it!

Paul

**Comment by skttl** [comment]
No worries - been there done that 😂

**Comment by skttl** [comment]
HI @prgriffithsdev 

I know what your problem was. Your browser caches the view for the directive (/App_Plugins/Our.Umbraco.Matryoshka/directives/matryoshka-tabbed-content.html). This part is not handled by client dependency, but I think you can get the current version and append that as a query string parameter, to bust the browser cache.

I'll reopen, so I can get it fixed :)

**Comment by prgriffithsdev** [comment]
Hi @skttl , 

Thanks for letting me know i wasn't going crazy :). 

Ill keep an eye for the fix and pull it in when you get around to solving it :). All is working fine since it resolved the other day

Thank you

**Comment by skttl** [comment]
Fixed in 8e339da
